src/server/utils/system_utils.js

```diff
@@ -23,6 +23,7 @@ function system_in_maintenance(system_id) {
 
 
 // returns the percent of quota used by the bucket
+// TODO remove the function
```

Comment:
Should this TODO be here? 
If so, why not just remove the function and remove the TODO?

Reply:
Thanks